### DISCUSSION TITLE
Unable to change parameters thru web interface

### DISCUSSION THREAD

**Discussion by FGM33** [emsesp/EMS-ESP32#1066] [Q&A]
Hi,
I've just installed a BBQKees gateway .
Dashboard is perfect BUT whereas I can see all parameters,  editing/chanhging any of them doesn't produce any result . 
The msge "write command sent" shows up but nothing happens .
Am I missing sumthing ?
Rgds

**Comment by proddy** [comment]
maybe. Can you read https://emsesp.github.io/docs/Support/ and attach the support info.
